index.html

```diff
@@ -493,8 +493,98 @@ <h4>Update Status</h4>
           data-api-endpoint="post /credentials/status"></div>
       </section>
 
+    </section>
+
+    <section>
+      <h3>Verifying</h3>
+      <p>
+The following APIs are defined for verifyig a Verifiable Credential:
+      </p>
+
+      <table class="simple api-summary-table"
+        data-api-path="/credentials/verify /presentations/verify"></table>
+
       <section>
-        <h4>Automation via Workflows</h4>
+        <h4>Verify Credential</h4>
+        <p>
+        </p>
+
+        <div class="api-detail"
+          data-api-endpoint="post /credentials/verify"></div>
+      </section>
+
+      <section>
+        <h4>Verify Presentation</h4>
+        <p>
+        </p>
+
+        <div class="api-detail"
+          data-api-endpoint="post /presentations/verify"></div>
+      </section>
+
+    </section>
+
+    <section>
+      <h3>Presenting</h3>
+      <p>
+The following APIs are defined for presenting a Verifiable Credential:
+      </p>
+
+      <table class="simple api-summary-table"
+        data-api-path="
+          /credentials/derive /presentations/prove
+          /presentations/available /presentations/submissions"
+        ></table>
+
+      <section>
+        <h4>Derive Credential</h4>
+        <p>
+        </p>
+
+        <div class="api-detail"
+          data-api-endpoint="post /credentials/derive"></div>
+      </section>
+
+      <section>
+        <h4>Prove Presentation</h4>
+        <p>
+        </p>
+
+        <div class="api-detail"
+          data-api-endpoint="post /presentations/prove"></div>
+      </section>
+
+      <section>
+        <h4>Presentation Availability</h4>
+        <p>
+        </p>
+
+        <div class="api-detail"
+          data-api-endpoint="post /presentations/available"></div>
+      </section>
+
+      <section>
+        <h4>Submit Presentation</h4>
+        <p>
+        </p>
+
+        <div class="api-detail"
+          data-api-endpoint="post /presentations/submissions"></div>
+      </section>
+
+    </section>
+
+    <section>
+      <h3>Workflows</h3>
+      <p>
+The following APIs are defined for executing Verifiable Credential workflows.
+      </p>
+
+      <table class="simple api-summary-table"
+        data-api-path="
+          /workflows/{name}/start
```

Comment:
dislike using name parameter here, per our previous resolutions on naming conventions: https://restfulapi.net/resource-naming/

I would prefer a controller style endpoint for this:

```
POST /workflows/initiate
POST /workflows/submit
```

^ Notice that these are nearly identical to the existing holder exchange endpoints.

Reply:
if we are going to include `id` in the route... its domain becomes limited to URL safe IDs... aka not other URLs..

When you are the operator of the resource server, and you assign the ID, this is a safe assumption... but I have seen this create endless pain that is easily avoided by proper use of the `controller` naming convention.

also consider the current "status" endpoints... they pass the `credentialId` via post body... mostly because it can be a URL.

Reply:
A couple of points: 

1) It sounds like @mprorock is challenging the "controller style endpoint" pattern that the group provisionally agreed to. If that is the case, I agree with @mprorock -- I've never liked the "controller style endpoint" pattern and we should see if the group has new opinions on this now that we have some implementation experience under our belt.

2) I agree with @mprorock's comment on the `POST /workflows/{name|ID}` or `PUT /workflows/{name|ID}` pattern.

3) I wrote this up in this way because when I asked the question to the group, @OR13 wanted it to be defined in this way rather than it be a "pattern" that can be used. That is, I'd rather /not/ define ONE endpoint to start all workflows or accept all presentations. Developers should be free to choose any endpoint to act in this way. I'm not going to argue that here, though. I just needed to get this down in a way that @OR13 felt like we could have a discussion over it. I think focusing on the name/structure of the URL is probably too deep in the weeds at this point of the analysis.

4) I agree that there is a debate to be had about whether or not `{id}` belongs in the URL or if it belongs in a POST body. One argument to allow it in the URL is for when browsers are invoked by native apps, where it is not possible to perform a POST. 

In short, there is a lot to discuss here, so I've raised an issue to track this concern: https://github.com/w3c-ccg/vc-api/issues/256